cmd/remote/games/tracker.go

```diff
@@ -3,14 +3,15 @@ package games
 import (
 	"bytes"
 	"encoding/json"
+	"net/http"
+	"os"
+	"path/filepath"
+
 	"github.com/wizzomafizzo/mrext/cmd/remote/websocket"
 	"github.com/wizzomafizzo/mrext/pkg/config"
 	"github.com/wizzomafizzo/mrext/pkg/mister"
 	"github.com/wizzomafizzo/mrext/pkg/service"
 	"github.com/wizzomafizzo/mrext/pkg/tracker"
-	"net/http"
-	"os"
-	"path/filepath"
```

Comment:
this is just VSCODE having opinions on import order. If i find the option i can disable it

Reply:
No problem. I use vs code now so you can leave it. It was a Goland thing before